huaweicloud/services/acceptance/vpn/resource_huaweicloud_vpn_customer_gateway_test.go

```diff
@@ -87,6 +201,9 @@ func TestAccCustomerGateway_basic(t *testing.T) {
 				ResourceName:      rName,
 				ImportState:       true,
 				ImportStateVerify: true,
+				ImportStateVerifyIgnore: []string{
+					"route_mode",
```

Comment:
you didn't input `route_mode`, and didn't set when read, why you have to ignore it?